eel/edge.py

```diff
@@ -1,20 +1,48 @@
-import platform
+import os
 import subprocess as sps
 import sys
 from typing import List
 
 from eel.types import OptionsDictT
 
-name: str = 'Edge'
+name: str = "Edge"
```

Comment:
Although admittedly there are a few inconsistencies, the eel package generally prefers single quotes for strings. I think it would be good if you could change these back throughout.